async-kafka/apps/emergency-location-service/src/main/java/com/redhat/training/emergency/route/EmergencyLocationRouteBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.training.emergency.route;
+
+import com.redhat.training.emergency.model.Location;
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.model.dataformat.BindyType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmergencyLocationRouteBuilder extends RouteBuilder {
+
+    @Override
+    public void configure() throws Exception {
+        onException(Exception.class).setBody(constant("errorOccured")).maximumRedeliveries(0).continued(true);
+
+        from("file://../resources/data?fileName=locations.csv&noop=true")
+        .routeId("emergency-location-route")
+        .transform(body())
+        .unmarshal()
+        .bindy(BindyType.Csv, Location.class)
+        .split(body())
+        // TODO: Produce to Location data to Kafka instead of writing it to the database
+        .setBody(simple("insert into locations values('${body.latitude}','${body.longitude}')"))
+        .to("jdbc:dataSource")
+        .to("direct:logger");
+
+
+        //TODO: Consume Location data from `locations` Kafka topic and write it to the database
+        //from([KAFKA CONSUMER ENDPOINT])
+        //.routeId("kafka-consumer-route")
+        //[WRITE TO THE DATABASE BY USING JDBC]
+        //.to("direct:logger");
```

Comment:
I'm not a fan of this way of showing the solution, but I'm open to see what people thinks on the pilot.